assets/panel/debugger.properties

```diff
@@ -603,6 +603,10 @@ sourceFooter.codeCoverage=Code coverage
 # with the current cursor line and column
 sourceFooter.currentCursorPosition=(%S, %S)
 
+# LOCALIZATION NOTE (sourceFooter.currentCursorPosition.tooltip): Text associated
+# with the current cursor line and column
+sourceFooter.currentCursorPosition.tooltip=(Line %S, column %S)
```

Comment:
One thing that I missed in the line above: we should try to explicitly use ordered parameters when there's more than one.

```
 sourceFooter.currentCursorPosition.tooltip=(Line %1$S, column %2$S) 
```

Reply:
And should the first letter `c` in `column` be capitalized?